instrumentation/http-tests/src/main/java/brave/test/http/ITHttp.java

```diff
@@ -154,23 +148,7 @@ protected Tracer tracer() {
 
   protected Tracing.Builder tracingBuilder(Sampler sampler) {
     return Tracing.newBuilder()
-      .spanReporter(s -> {
```

Comment:
NOTE: one sad side-effect is this removes our ability to do span leak detection this way.. someone clever can figure another one.

Reply:
actually this only helps with third party impls anyway as it proved nothing when using our middleware HttpHandler